python/arcticdb/version_store/library.py

```diff
@@ -1039,6 +1039,8 @@ def write(
                 lib.write(symbol, data) # The data will be successfully written
             Please refer to https://docs.arcticdb.io/latest/notebooks/arcticdb_demo_recursive_normalizers for more details
             of this feature.
+            Please check https://docs.arcticdb.io/latest/runtime_config/#versionstorerecursivenormalizermetastructurev2 for the plan of
```

Comment:
I'd add a sentence describing the main benefit of V2 metastruct  here (and remove it when v7 is released)